posthog/temporal/tests/test_squash_person_overrides_workflow.py

```diff
@@ -645,7 +644,6 @@ async def test_squash_events_partition_with_older_overrides(
 
 
 @pytest.mark.django_db
-@pytest.mark.asyncio
```

Comment:
We set `pytestmark` for all tests so these are not needed on individual functions anymore.